cross/arm-none-eabi-binutils/Portfile

```diff
@@ -3,9 +3,13 @@
 PortSystem          1.0
 PortGroup           crossbinutils 1.0
 
-crossbinutils.setup arm-none-eabi 2.33.1
+crossbinutils.setup arm-none-eabi 2.34
 maintainers         nomaintainer
 
+checksums           rmd160  e99f517407503cec1042a2e824b7c1f8f0fca4b8 \
```

Comment:
I went ahead and bumped the all the crossbinutils that seemed to have been kept relatively up-to-date, though I've only tested arm-none-eabi because that's the only one I have use for currently.

Reply:
Sure, there are some special ports which might be problematic to update as they require some more extensive patching, else they would have been updated in the last round already.